apps/old/widget/components/UploadField.jsx

```diff
@@ -13,9 +13,11 @@ const uploadFile = (files) => {
     })
     .then((res) => {
       setImg(res.body.cid);
-      props.setImage({
-        ipfs_cid: res.body.cid,
-      });
+      if (props.onChange) {
+        props.onChange({
+          ipfs_cid: res.body.cid,
+        });
+      }
```

Comment:
also this one @elliotBraem

Reply:
Yes also keep this one, another reason for the break